tox.ini

```diff
@@ -12,7 +12,7 @@ deps =
     django19: Django>=1.9,<1.10
     django110: Django>=1.10,<1.11
     django111: Django>=1.11,<1.12
-    django20: Django>=2.0,<2.1
+    django20: Django>=2.0,<2.2
```

Comment:
This is wrong, we need to test both Django 2.0 as well as 2.1. I.e.:

```
django20: Django>=2.0,<2.1
django21: Dango>=2.1,<2.2
```

Also make sure to update the envlist with `django21`

Reply:
This doesn't appear to have changed on github, did you push your changes?